### DISCUSSION TITLE
UE 5.1 Version - Missing Folders

### DISCUSSION THREAD

**Discussion by Arynaine** [Sixze/ALS-Refactored#240] [Q&A] [answered]
Hi!

Now this might be a mighty easy question, but I cannot get this to work in UE5.1.

Steps Taken:
Create project
Copy ALS-Refactored into Plugins folder
Merged DefaultEngine.ini and DefaultInput.ini files
Changed .uplugin file to 5.1
Removed Saved, Intermediate and Binary Files
Regenrate Vs files
Opened and Rebuilt project

At this point, it builds fine, and I can see that the plugin is enabled - but there is no files for me to use in the Content Editor.

What am I doing wrong? 

Thank you!

**Comment by juddisjudd** [comment, marked as answer]
Definitely works in 5.1, make sure you "Show Plugin Content"
![image](https://user-images.githubusercontent.com/85649796/218133450-1446c7b6-d0bc-4455-9ff5-3db0575b7d73.png)

Also run the "BuildPlugin.bat" and in the bat change the EngineVersion to 5.1 which will then create a "build" folder, rename that folder to ALS or something and put that in your Plugins folder (works in BP and C++ projects).

**Reply by Arynaine** [reply]
It was just the damn cogwheel I missed. Thank you for the answer and have a great day!
